### ISSUE TITLE
How to generate mars-small128.ckpt file for new dataset(not pedestrian)

### ISSUE THREAD

**Issue by DarshanaK** [nwojke/deep_sort#2]

**Comment by nwojke** [comment]
The model was trained on a person re-identifcation dataset using [TensorFlow-Slim](https://github.com/tensorflow/models/blob/master/inception/inception/slim/README.md). Our training code is not yet published, but may follow in the future.

For now, have a look at the `_network_factory` in `generate_detections.py` if you want to re-train the model from scratch. However, If you want to apply the method to other object classes, you may want to swap the entire network architecture.

**Comment by Projetstepsenseeiht** [comment]
Is it possible to use directly the network in order to track people in other set (i.e not motset) .
Thanks!

**Comment by nwojke** [comment]
I have written up a response in [Issue 4](https://github.com/nwojke/deep_sort/issues/4). Let's move the discussion there.

**Comment by WesleyZhang1991** [comment]
Hi, I train a similar network with mars 1261 dataset in mxnet. However, the result is not as good. I have several questions.

I find that there is a background class in mars dataset and the number of each class is unbalanced. Do you consider the background as a class?  Also Is there any data balance?
When considering data augmentation, I notice that color manipulation is used. Do you also introduce random crop?
What is your evaluation metric for the feature extractor? The person reid metric or tracking performance?
Thanks
